src/aap_eda/core/migrations/0029_alter_activation_status_alter_eventstream_status_and_more.py

```diff
@@ -0,0 +1,114 @@
+import django.db.models.deletion
+from django.db import migrations, models
+
+import aap_eda.core.enums
+
+
+def populate_rulebook_process_queue(apps, schema_editor):
+    RulebookProcessQueue = apps.get_model(  # noqa: N806
+        "core", "RulebookProcessQueue"
+    )
+    RulebookProcess = apps.get_model("core", "RulebookProcess")  # noqa: N806
+    for running_process in RulebookProcess.objects.filter(
+        status="RUNNING",
+    ).all():
+        RulebookProcessQueue.objects.create(
```

Comment:
@Alex-Izquierdo what would happen post upgrade when there won't be a queue called activation since we now have per node queue name. Will these rulebooks get stuck because after the upgrade there won't be a queue called activation. As part of the upgrade do all activations have to be stopped and they will get restarted automatically when the restart policy is applied on it.

Reply:
The installer will kill the activations if it is converting a node. When the pod is not running it is restarted (missing container policy)